site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -17,6 +17,7 @@ The remainder of this article describes how to enable multi-cluster allocation.
 ## Define Cluster Priority
 
 {{< ghlink href="pkg/apis/multicluster/v1/gameserverallocationpolicy.go" >}}GameServerAllocationPolicy{{< /ghlink >}} is the CRD defined by Agones for setting multi-cluster allocation rules. In addition to cluster priority, it describes the connection information for the target cluster, including the game server namespace, agones-allocator endpoint and client K8s secrets name for redirecting the allocation request. Here is an example of setting the priority for a cluster and it's connection rules. One such resource should be defined per cluster. For clusters with the same priority, the cluster is chosen with a probability relative to its weight.
+The lower the `priority` number is the higher is the corresponding cluster's priority for allocating game servers.
```

Comment:
Nit: This doesn't quite read correctly. How about:


In addition to cluster priority, it describes the connection information for the target cluster, including the game server namespace, agones-allocator endpoint and client K8s secrets name for redirecting the allocation request. Game servers will be allocated from clusters with the lowest `priority` number. If there are no available game servers available in clusters with the lowest `priority` number, they will be allocated from clusters with the next lowest `priority` number. For clusters with the same priority, the cluster is chosen with a probability relative to its weight. 

Here is an example of setting the priority for a cluster and it's connection rules. One such resource should be defined per cluster.

Reply:
Good suggestion. Thanks!